.github/workflows/issue.yml

```diff
@@ -12,18 +12,24 @@
 name: Issues Workflow
 on:
   issues:
-      types: ['opened']
+    types: ["opened"]
 jobs:
   Opened-issue-label:
     name: Adding Issue Label
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v4
```

Comment:
Please don' t rename this file. It's a standard name across all our repositories.

Reply:
Yes, I understand. 
![image](https://github.com/user-attachments/assets/47b8654e-4e86-4fe2-8be6-98bfd5f7ed4c)